### PR TITLE
feat(suspicious_login): Add "more info" button to email notifications

### LINKED ISSUE #801: Add button to email notifications to get more info about the suspicious ip
Hi!
I think it's good that the systems checks (and learns) about IP/Userid combinations.

- Nevertheless I think the message should be more helpful to allow a click on

https://www.whatismyip.com/ip-address-lookup/  + IP Address to quickly show the location - that's what the warning is advising.
At least it could give a hint, but may be not helpful for user-nomads with many different IP's

- it would also be good if the admin (or user) is able to see the table content.

### PR DESCRIPTION
**Pull request by joshtrichards**
Fixes #801

Relies on existing app config parameter already used for toggling in-app notifications.

### CODE REVIEW

**Comment by joshtrichards** [comment]
Tested. Works. :)

File: lib/Listener/LoginMailListener.php [hunk 1]
```diff
@@ -40,21 +42,25 @@ class LoginMailListener implements IEventListener {
...
 	private $mailer;
+	
 	/** @var IUserManager */
 	private $userManager;
+	
 	/** @var IL10N */
 	private $l;
+	
+	/** @var IConfig */
+	protected $config;
```
Comment (st3iny) [inline lib/Listener/LoginMailListener.php:56]:
```suggestion
	private IConfig $config;
```

**Review by st3iny — COMMENTED** [review]
Looks good otherwise. I did not test.

File: lib/Listener/LoginMailListener.php (hunk 1 above)
Comment (st3iny) [reply lib/Listener/LoginMailListener.php:56]: Let's not get caught up in minor details.

**Review by st3iny — APPROVED** [review]
Tested and works. Thank you very much!

File: lib/Listener/LoginMailListener.php (hunk 1 above)
Comment (joshtrichards) [reply lib/Listener/LoginMailListener.php:56]: Thanks for the note. I'll try to keep in mind next time. :)

**Pull request merged**